lib/stream.js

```diff
@@ -48,15 +48,18 @@ class ServerSentEvent {
  * @param {string} config.url The URL to connect to.
  * @param {typeof fetch} [config.fetch] The URL to connect to.
  * @param {object} [config.options] The EventSource options.
+ * @param {boolean} [config.options.useFileOutput] Whether to use the file output stream.
  * @returns {ReadableStream<ServerSentEvent> & AsyncIterable<ServerSentEvent>}
  */
 function createReadableStream({ url, fetch, options = {} }) {
+  const { useFileOutput = false, headers = {}, ...initOptions } = options;
```

Comment:
What do you think to just always returning a `FileOutput` instance from the stream? The url is available via the object anyway, so there's no breaking changes here.

Reply:
Yeah, that sounds good to me. So something this, then?

```suggestion
  const { useFileOutput = true, headers = {}, ...initOptions } = options;
```